spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -16,15 +15,13 @@
   end
 
   before(:each) do
-    allow(Signet::OAuth2::Client).to receive(:new)
-      .and_return(fake_auth_client)
-    allow(fake_auth_client).to receive(:fetch_access_token!)
-    allow(fake_auth_client).to receive(:access_token)
+    allow(api_client).to receive(:auth_token)
```

Comment:
Since auth token has an entire module and spec dedicated to it. I thought it would be ok to mock this out since these tests  shouldn't rely on the auth token working correctly. We would only want it to fail if it was a problem with that class not the auth token.

Reply:
In this case `auth_token` was calling `fetch_auth_token`, which I would have preferred you mock since that was coming from the helper.

Now you've removed `auth_token` completely so you should be able to remove this mock, right?

Reply:
Yes that is right. Will remove the mock.